### DISCUSSION TITLE
Is there any way to reduce swipe sensitivity?

### DISCUSSION THREAD

**Discussion by assimovt** [software-mansion/react-native-gesture-handler#1668] [Q&A]
I've noticed that swiping is too sensitive, so touching Swipeable just a bit activates it right away. Is there any way to reduce sensitivity, for instance, let the user swipe more to open the row? I've noticed there are `friction` and `overshootFriction` props, but they seem to be unrelated. Any ideas? Thanks!

**Comment by pierroo** [comment]
Did you find an answer to your question since @assimovt ? :)
I am facing the same issue.
(too sensitive, even more obvious when in a flatlist where you scroll and it mistakenly triggers horizontal swipe)

**Comment by thesoicalapp91** [comment]
Hi All - this seams to work for me - (On android only, the default code works fine on ios)

1. Go into Swiper node modules:
- File: node_modules => react-native-swiper => src => index.js

2. Find and comment out the below line - (current behaviour):
- // index = parseInt(index + Math.round(diff / step))

3. Create custom index/sensitivity
-   const threshold = 0.5; // increase this to make it less sensitive
    if (Math.abs(diff) > step * threshold) {
      index = index + (diff > 0 ? 1 : -1);
    }

Add the above code underneath the commented out code and before the : if (this.props.loop) {.....

seams to work for me.
